formulas/glance/configure.sls

```diff
@@ -74,3 +74,44 @@ glance_api_service:
     - enable: true
     - watch:
       - file: /etc/glance/glance-api.conf
+
+download_image_bakery:
+ cmd.run:
+     - name: curl https://gitlab.com/gacybercenter/image-bakery/-/raw/master/image_bake.py -o /tmp/image_bake.py
+     - creates: /tmp/image_bake.py
+
+/etc/openstack/clouds.yml:
+  file.managed:
+    - source: salt://formulas/common/openstack/files/clouds.yml
+    - makedirs: True
+    - template: jinja
+    - defaults:
+        password: {{ pillar['openstack']['admin_password'] }}
+        auth_url: https://dashboard.gacyberrange.org:5000/v3
```

Comment:
This should reference the pillar for crafting the URL rather than specifying our internal endpoint explicitly - we want this to be able to work "anywhere" and not just for us.